modules/simple_forms_api/app/controllers/simple_forms_api/v1/scanned_form_uploads_controller.rb

```diff
@@ -27,13 +27,41 @@ def upload_scanned_form
         render json: PersistentAttachmentVAFormSerializer.new(attachment)
       end
 
+      def upload_supporting_documents
+        unless Flipper.enabled?(:simple_forms_upload_supporting_documents, @current_user)
+          render json: { error: 'Feature not available' }, status: :not_found
+          return
+        end
+
+        attachment = PersistentAttachments::MilitaryRecords.new
+        attachment.form_id = params['form_id']
+        attachment.file = params['file']
+        raise Common::Exceptions::ValidationErrors, attachment unless attachment.valid?
+
+        processor = SimpleFormsApi::ScannedFormProcessor.new(attachment)
+        processed_attachment = processor.process!
+
+        render json: PersistentAttachmentVAFormSerializer.new(processed_attachment)
+      rescue SimpleFormsApi::ScannedFormProcessor::ConversionError,
+             SimpleFormsApi::ScannedFormProcessor::ValidationError => e
+        render json: { errors: e.errors }, status: :unprocessable_entity
```

Comment:
I notice this is `errors:` and the other is `error:`, I suppose the client is check for both?